README.md

```diff
@@ -62,6 +62,18 @@ Creates an SQS queue.
 ## Security Group
 Creates a security group with rules passed to it.
 
+## WAFv2 AWS Managed Rules
+Creates a set of AWS managed WAF rules to be associated with the given AWS resources.
+
+Provides a default set of AWS managed rules. The default rules are "free" and non-language and non-OS specific:
+* `AWSManagedRulesAmazonIpReputationList`
+* `AWSManagedRulesCommonRuleSet`
+* `AWSManagedRulesCommonRuleSet`
+
+Calling clients are can define whatever set of AWS managed rules are needed and are not restricted to the default rules.
```

Comment:
I'm not sure that `Calling clients are can define` is quite right.

Reply:
@MancunianSam How about something like this: `Calling modules can chose whatever set of AWS managed rules are most appropriate for the particular service and are not restricted to the default rules.`

Reply:
I'm not the best to ask about documentation but that seems sensible. I'd use `choose` instead of `chose` though.